src/main/java/com/connectfour/Display.java

```diff
@@ -0,0 +1,24 @@
+package com.connectfour;
+
+import java.io.PrintStream;
+
+public class Display implements Displayable{
+
+    // for Strings
+    public void print(String message){
+        System.out.print(message);
+    }
+    public void println(String message){
+        System.out.println(message);
+    }
+    // for chars
+    public void print(char message){
+        System.out.print(message);
+    }
+    public void println(char message){
+        System.out.println(message);
+    }
+
```

Comment:
I didn't see any unit tests for Display. In any case, how will you test Display if you don't use dependency injection?  See this example:  https://github.com/kudukacar/server/blob/master/src/main/java/infrastructure/Logger.java

Reply:
Good point, originally had it like that, then changed it. Will change it back